src/components/navigation/Footer.tsx

```diff
@@ -85,7 +111,37 @@ const useStyles = makeStyles((theme: Theme) => ({
     color: theme.palette.getContrastText(theme.palette.colors.footer),
   },
   themeSettingsIcon: {
-    fontSize: 30,
+    fontSize: 36,
+  },
+  attribute: {
+    textAlign: 'center',
+    marginBottom: theme.spacing(2),
+  },
+  attributeName: {
+    textTransform: 'uppercase',
+    color: theme.palette.common.white,
+    fontWeight: 600,
+  },
+  logo: {
+    minWidth: '250px',
+    width: '46%',
+    height: 'auto',
+    [theme.breakpoints.down('sm')]: {
+      minWidth: '200px',
+    },
+  },
+  divider: {
+    margin: theme.spacing(1, 0, 2),
+    backgroundColor: '#fff',
+    height: 1,
+    width: 175,
+  },
+  soMeIcon: {
+    width: 38,
+    transition: 'all .15s',
```

Comment:
Bedre å spesifisere hva som skal animeres hvis det er mulig
```suggestion
    transition: 'transform .15s',
```